### ISSUE TITLE
Suggestion: Document ko.utils on website

### ISSUE THREAD

**Issue by webketje** [knockout/knockout#1742]
It would be tremendously useful if Knockout's utility functions were documented on the website, especially for people not using jQuery (eg for `ko.utils.registerEventHandler`), not aware of how to map JSON data *without* the mapping plugin (`ko.utils.arrayMap`), or surprised to find code like on the `select` binding doc. page, where `optionsAfterRender` mentions a function `ko.utils.applyBindingsToNode`. There are other examples.

Personally I'd learnt about  `utils` functions one by one in SO posts, reading [R. Niemeyer's article](http://www.knockmeout.net/2011/04/utility-functions-in-knockoutjs.html) and finally, sifting through the source code.

**Comment by brianmhunt** [comment]
Agreed. I'm mulling the documentation processes - it would be great if it were centralized and up-to-date with many of the diverse techniques one will find on SO, blogs, and in the code.

Cheers
